index.js

```diff
@@ -199,6 +203,7 @@ export const revoke = async (
     method: 'POST',
     headers: {
       'Content-Type': 'application/x-www-form-urlencoded',
+      Authorization: `${includeBasicAuthorization ? `basic ${basicAuth}` : ''}`,
```

Comment:
I'd prefer creating headers as a constant above and adding the auth header optionally:
```
const headers = {
  'Content-Type': 'application/x-www-form-urlencoded',
};

if (includeBasicAuthorization) {
  headers.Authorization =`basic ${base64.encode(`${clientId}:${clientSecret}`)}`,
}
```

This way you only have one conditional statement and it's a bit easier to follow

Reply:
done

Reply:
Thank you! I've just been thinking about this PR, and would really like to avoid adding a dependency if possible (I'm planning on getting rid of `invariant` as well).

What do you think about instead of `includeBasicAuthorization`, we could pass in a `customHeaders` param (we already do something similar for the `authorize` call). This way, the logic for the base 64 encoding would live in the app code, rather than the library 🤔

Reply:
I had a similar idea at first but I thought we could make simpler for the app code since we already have clientId & clientSecret in the config (I guess the native library is doing the same), but the down side of it we would add an extra dependency.

let me know what you think I will be happy to create a new pull request for your approach.

Reply:
Alright, let's just leave it as 👍